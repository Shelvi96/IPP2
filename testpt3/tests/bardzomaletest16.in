NEW ID

;728>48:079
:9>33
;5;>457:9:
47649>;
::>9;2707
46>1
5561733:8:>0;638
5595755>810221808
;;>9;653
371;06>9567;684
358>1965
44718453>8739507041
@2
@522;0945
@0568933;03210790;24;848;6;;778
@81383642:780739;63790;:;69259
@:5415
@;99:89:3
@;6;621
@1;9;0;5923668457165
@3758:
@0
@253381
@040:6384;0
@:6
@883713:17770:3
