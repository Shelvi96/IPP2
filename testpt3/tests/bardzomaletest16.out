0
0
1056915999224555459
385195606216261632
0
0
0
3704426
0
0
0
0
0
7
